### ISSUE TITLE
GETTING_STARTED.md does not exist on RubyDoc

### ISSUE THREAD

**Issue by BerkhanBerkdemir** [thoughtbot/factory_bot#1172]
Hello,

I was looking the documentation, but GETTING_STARTED.md which is on README.md returns not found on RubyDoc.

I want to ask that because you have GETTING_STARTED.md on the codebase; however, you redirect to RubyDoc.

**Comment by composerinteralia** [comment]
Nice catch. It looks like I should have included .yardopts in https://github.com/thoughtbot/factory_bot/pull/1167. Also, I think .yardopts should have included NAME.md. I suspect we have a similar problem in factory_bot_rails. Any interest in opening a PR?

**Comment by BerkhanBerkdemir** [comment]
@composerinteralia, so, you want to add something like that, right?

[`.yardopts`](https://github.com/thoughtbot/factory_bot/blob/dacc167803e87fc1433f91493ff8cf0b8aa583fe/.yardopts)
```diff
CONTRIBUTING.md
+ NAME.md
LICENSE
```

[`factory_bot.gemspec`](https://github.com/thoughtbot/factory_bot/commit/fc451872b9b4e7e17b1d19e8e6f27d918acd3ef8#diff-ff1a92cd12d8179e487b79302e7e9316R15)
```diff
+ %w[CONTRIBUTING.md GETTING_STARTED.md LICENSE NAME.md NEWS README.md .yardopts]
```

**Comment by composerinteralia** [comment]
Yeah, I think so

**Comment by BerkhanBerkdemir** [comment]
Hi @composerinteralia,

I created a PR. Could you check it out?

**Comment by composerinteralia** [comment]
Fixed in #1175
